FreeRTOS/FreeRTOS-Kernel
# Fix cmake example errors

Description
-----------
This PR fixes the _'int' to 'BaseType_t'_ and _'int' to 'UBaseType_t'_ conversion errors in FreeRTOS+Kernel Source files.

Test Steps
-----------
Compiled the [cmake example](https://github.com/FreeRTOS/FreeRTOS-Kernel/tree/main/examples/cmake_example) without any warnings/errors.

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
